File: Python x EDAII/Ordenamiento/SlowSort.py
class SlowSort:

	arr = []	# Arreglo de elementos

	def __init__(self, arr):
		self.arr = arr

	def slowSort(self, arr, inf, sup):
		'Algoritmo de ordenamiento.'
		if inf >= sup:
			return

		m = int((inf+sup)/2)
		self.slowSort(arr,inf,m)	# Subarreglo izquierdo
		self.slowSort(arr,m+1,sup)	# Subarreglo derecho

		if arr[sup] < arr[m]:
			# Intercambio
			temp = arr[sup]
			arr[sup] = arr[m]
			arr[m] = temp

		self.slowSort(arr,inf,sup-1)

# Ejecutando objetos
programa = SlowSort([50,17,48,23,8,60])
programa.slowSort(programa.arr,0,len(programa.arr)-1)
print(programa.arr)

File: EDAII x Java/Ordenamiento/SlowSort.java
public class SlowSort{

	int i,n;
	int[] arr = {50,17,48,23,8,60};

	public static void main(String[] args){
		SlowSort ss = new SlowSort();
		ss.slowSort(ss.arr, 0, ss.arr.length-1);  //Ordenamiento SlowSort
		ss.n = ss.arr.length;

		for(ss.i=0; ss.i<=ss.n-1; ss.i++){
			System.out.print(ss.arr[ss.i] + ", ");
		}
		System.out.println("\n");
	}

	public void slowSort(int[] A, int inf, int sup){
		if (inf >= sup){
			return;
		}

		int m = (inf+sup)/2;
		slowSort(A,inf,m);	// Subarreglo izquierdo
		slowSort(A,m+1,sup); // Subarreglo derecho

		if (A[sup] < A[m]){
			// Intercambio
			int temp = A[sup];
			A[sup] = A[m];
			A[m] = temp;
		}

		slowSort(A,inf,sup-1);
	}

}